src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./navBar.module.css";
+import Typography from "@mui/material/Typography";
+import { Link } from "@mui/material";
+
+export function NavBar() {
+  return (
+    <div className={styles.navBar}>
+      <div>
+        <Link href={"/"}>
+          <Typography variant="homeLink">bookify</Typography>
+        </Link>
+      </div>
+      <div className={styles.linksList}>
+        <Link href={"/"} variant="link">
+          <Typography variant="aboutUsLink">about us</Typography>
+        </Link>
+        <Link href={"/"} variant="link">
```

Comment:
Use link from router not the one from mui. Please create empty pages and add them to routing so we can practice the router skills.